### DISCUSSION TITLE
Renovate Mostly Stopped Updating Lock Files Today

### DISCUSSION THREAD

**Discussion by Kurt-von-Laven** [renovatebot/renovate#20920] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

We are using the Forking Renovate GitHub App. Renovate has continued updating package files (e.g., `.tool-versions`, `package.json`, `pyproject.toml`) as reliably as ever. With one exception, Renovate was unable to update lock files (e.g., `poetry.lock` and `yarn.lock`) on all of our repositories today. Only a few bumps were completely successful:

- All bumps of [poetry-core](https://pypi.org/project/poetry-core/), e.g. ScribeMD/docker-cache#430, as these don't affect `poetry.lock`.
- A bump of [@typescript-eslint/eslint-plugin](https://www.npmjs.com/package/@typescript-eslint/eslint-plugin) and [@typescript-eslint/parser](https://www.npmjs.com/package/@typescript-eslint/parser) in ScribeMD/docker-cache#429, but not other contemporaneous bumps of Yarn dependencies, such as ScribeMD/docker-cache#426 and ScribeMD/docker-cache#428.
- A bump of [Node.js](https://github.com/asdf-vm/asdf-nodejs) and [@types/node](https://www.npmjs.com/package/@types/node) in ScribeMD/docker-cache#427.

Here is an example of the error message from a failed bump of a Yarn dependency, namely ScribeMD/docker-cache#426:

### ⚠ Artifact update problem

Renovate failed to update an artifact related to this branch. You probably do not want to merge this PR as-is.

♻ Renovate will retry this branch, including artifacts, only when one of the following happens:

 - any of the package files in this branch needs updating, or 
 - the branch becomes conflicted, or
 - you click the rebase/retry checkbox if found above, or
 - you rename this PR's title to start with "rebase!" to trigger it manually

The artifact failure details are included below:

##### File name: yarn.lock

```
installing v2 tool node v18.15.0
linking tool node v18.15.0
node: v18.15.0 /usr/local/bin/node
npm: 9.5.0  /usr/local/bin/npm
Installed v2 /usr/local/buildpack/tools/v2/node.sh in 4 seconds
skip cleanup, not a docker build: 3f9b5a8bc268
installing v2 tool corepack v0.17.0
linking tool corepack v0.17.0
0.17.0
Installed v2 /usr/local/buildpack/tools/v2/corepack.sh in 2 seconds
skip cleanup, not a docker build: 3f9b5a8bc268
➤ YN0000: ┌ Resolution step

```

Here is an example of the error message from a failed bump of our Poetry lock file, namely ScribeMD/template-repo#114:

### ⚠ Artifact update problem

Renovate failed to update an artifact related to this branch. You probably do not want to merge this PR as-is.

♻ Renovate will retry this branch, including artifacts, only when one of the following happens:

 - any of the package files in this branch needs updating, or 
 - the branch becomes conflicted, or
 - you click the rebase/retry checkbox if found above, or
 - you rename this PR's title to start with "rebase!" to trigger it manually

The artifact failure details are included below:

##### File name: poetry.lock

```
installing v2 tool python v3.11.2
linking tool python v3.11.2
Python 3.11.2
pip 23.0.1 from /opt/buildpack/tools/python/3.11.2/lib/python3.11/site-packages/pip (python 3.11)
Installed v2 /usr/local/buildpack/tools/v2/python.sh in 11 seconds
skip cleanup, not a docker build: 6861a5934fd9
installing v2 tool poetry v1.4.0


```

Here is an example from a failed bump of our Poetry and Yarn lock files, namely ScribeMD/docker-cache#431:

### ⚠ Artifact update problem

Renovate failed to update artifacts related to this branch. You probably do not want to merge this PR as-is.

♻ Renovate will retry this branch, including artifacts, only when one of the following happens:

 - any of the package files in this branch needs updating, or 
 - the branch becomes conflicted, or
 - you click the rebase/retry checkbox if found above, or
 - you rename this PR's title to start with "rebase!" to trigger it manually

The artifact failure details are included below:

##### File name: poetry.lock

```


```

##### File name: yarn.lock

```
installing v2 tool node v18.15.0

```

### What you tried so far.

I looked through the issue tracker, and the most similar issue I could find was #18690, but as no reproduction was provided, it couldn't be resolved. I checked https://developer.mend.io/, but received a 500 error. We didn't make any recent changes to our Renovate config. My instinct is that this may be a Renovate ops issue, hopefully limited to Forking Renovate.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Hi, I have been able to reproduce the problem and we are looking into it. How long ago was it when you had the last successful yarn PR?

**Reply by Kurt-von-Laven** [reply]
Technically, nine hours ago: ScribeMD/docker-cache#429. Given that that was the only successful PR around that time that touched `yarn.lock` though, I took it as a sign that the issue was not quite 100% consistent. ScribeMD/docker-cache#418 updated `yarn.lock`, and ScribeMD/slack-templates#322 updated `poetry.lock` four days ago without issue.

**Reply by Kurt-von-Laven** [reply]
Did you open ScribeMD/docker-cache#432 to aid with your investigation? Definitely not a problem, but in case you didn't, I thought it might be helpful to be aware that the PR was not opened by any user action taken on our end.

**Reply by rarkins** [reply]
It was manually triggered, but accidentally, by me. Sorry and thanks for your understanding!

**Reply by Kurt-von-Laven** [reply]
Not a problem! Saves me a step 😂.

**Comment by rarkins** [comment, marked as answer]
The error appeared to be memory-related, so we have increased resources per Renovate worker and the problem now disappears in my reproduction repo. Thanks for reporting!

**Reply by Kurt-von-Laven** [reply]
Thank you for the speedy fix! Renovate is working again on our repos as well.
